tox.ini

```diff
@@ -1,8 +1,16 @@
 [tox]
-envlist = pypy, pypy3, py27, py34, py35, py36
+envlist = pypy, pypy3, py27, py34, py35, py36, flake8
 
 [testenv]
 deps = -rrequirements-dev.txt
 commands =
     py.test sasstests.py
+
+[testenv:flake8]
+deps= flake8
+skip_install=True
+commands=
     flake8 .
```

Comment:
To elaborate, we really want to lint against each of the languages targetted as there are often differences in flake8 output depending on python version